elastic/integrations
# Rename config to policy and dataset to datastream

## What does this PR do?



## Checklist


## Author's Checklist


- [x] Rename config template to policy template
- [x] Rename dataset to data_stream
- [x] Bump up release versions of integrations
- [x] Make sure there are no "hacked" dependencies.

## How to test this PR locally

```
mage clean
mage build
cd testing/environments
docker-compose -f snapshot.yml build package-registry
docker-compose -f snapshot.yml up --force-recreate package-registry
```
package-registry should be exposed under http://localhost:8080 .



## Related issues


-

## Screenshots
